pkg/pmem-csi-driver/nodeserver.go

```diff
@@ -365,14 +362,7 @@ func (ns *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		}
 	}
 
-	// FIXME(avalluri): we shouldn't depend on volumecontext to determine the device mode,
```

Comment:
Apropos these FIXME notes: these have been in the code for a while. We should turn them into issues and that way actually track progress towards solving them.

At least we got rid of this one :smile: